iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m

```diff
@@ -31,27 +31,39 @@
 #import "AppDelegate.h"
 #import "ViewController.h"
 
+@interface OneSignalNotificationCenterDelegate: NSObject<UNUserNotificationCenterDelegate>
+@end
+@implementation OneSignalNotificationCenterDelegate
+
```

Comment:
some of the changes were covered on https://github.com/OneSignal/OneSignal-iOS-SDK/pull/687/files is that ok?